### PR TITLE
[Release] Add release 3.1.0

### PR DESCRIPTION
**Pull request by caishunfeng**

### CODE REVIEW

**Review by zhuangchong — APPROVED** [review]
LGTM.

File: download/zh-cn/download.md [hunk 1]
```diff
@@ -7,12 +7,12 @@
...
 | | | Total Binary Distribution| [[bin]](https://www.apache.org/dyn/closer.lua/dolphinscheduler/3.0.0/apache-dolphinscheduler-3.0.0-bin.tar.gz) [[asc]](https://downloads.apache.org/dolphinscheduler/3.0.0/apache-dolphinscheduler-3.0.0-bin.tar.gz.asc) [[sha512]](https://downloads.apache.org/dolphinscheduler/3.0.0/apache-dolphinscheduler-3.0.0-bin.tar.gz.sha512)|
-| Aug. 9th, 2022 | 3.0.0 | Total Source Code| [[src]](https://www.apache.org/dyn/closer.lua/dolphinscheduler/3.0.0/apache-dolphinscheduler-3.0.0-src.tar.gz) [[asc]](https://downloads.apache.org/dolphinscheduler/3.0.0/apache-dolphinscheduler-3.0.0-src.tar.gz.asc) [[sha512]](https://downloads.apache.org/dolphinscheduler/3.0.0/apache-dolphinscheduler-3.0.0-src.tar.gz.sha512)|
-| | | Total Binary Distribution| [[bin]](https://www.apache.org/dyn/closer.lua/dolphinscheduler/3.0.0/apache-dolphinscheduler-3.0.0-bin.tar.gz) [[asc]](https://downloads.apache.org/dolphinscheduler/3.0.0/apache-dolphinscheduler-3.0.0-bin.tar.gz.asc) [[sha512]](https://downloads.apache.org/dolphinscheduler/3.0.0/apache-dolphinscheduler-3.0.0-bin.tar.gz.sha512)|
```
Comment (zhongjiajie) [inline download/zh-cn/download.md:15]: why do we remove the verison 3.0.0 download link?

Comment (zhuangchong) [reply download/zh-cn/download.md:15]: I see this part is repeated in the documentation.

Comment (caishunfeng) [reply download/zh-cn/download.md:15]: Yes, it's repeated.
image

**Review by zhongjiajie — APPROVED** [review]

**Comment by zhongjiajie** [comment]
should not merge until https://github.com/apache/dolphinscheduler/pull/12227 merged

**Pull request merged**